docker-compose/kafka.yaml

```diff
@@ -94,28 +97,33 @@ services:
 
   schema-registry:
     mem_limit: 256m
-    image: confluentinc/cp-schema-registry:7.3.0
+    image: apicurio/apicurio-registry-kafkasql:2.4.1.Final
     hostname: schema-registry
     container_name: schema-registry
     depends_on:
       kafka:
         condition: service_healthy
     ports:
-      - "8081:8081"
+      - "8081:8080"
     environment:
-      SCHEMA_REGISTRY_HOST_NAME: schema-registry
-      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: "kafka:19092"
-      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:8081
+      ENABLE_KAFKA_SASL: true
+      CLIENT_ID: testId
+      CLIENT_SECRET: testSecret
+      OAUTH_TOKEN_ENDPOINT_URI: https://oauth.cheetah.trifork.dev/oauth2/token
+      KAFKA_BOOTSTRAP_SERVERS: kafka:19092
+      KAFKA_SECURITY_PROTOCOL: SASL_PLAINTEXT
+      REGISTRY_AUTH_ANONYMOUS_READ_ACCESS_ENABLED: true
+
     healthcheck:
-      test: nc -z localhost 8081 || exit -1
+      test: curl schema-registry:8080 || exit -1
       start_period: 15s
       interval: 5s
       timeout: 10s
       retries: 10
     profiles:
       - kafka
 
-
 networks:
   default:
     name: "cheetah-infrastructure"
+    external: true
```

Comment:
this means the developer needs to write:

`docker network create cheetah-infrastructure` before using docker compose.

Should be in readme.md